module.js

```diff
@@ -0,0 +1,136 @@
+const catchA = function () {
+    let name = '';
+    let start = false;
+    let score= 0 ;
+    let level=1 ;
+    let iphones=[];
+    let idCounter=1
+    let timer = 2
+   
+    const randomColor = function () {
+        let ranNum = function () {
+            return Math.floor(Math.random() * 255)
+        }
+        let color = `rgb(${ranNum()},${ranNum()},${ranNum()})`
+        return color
+    }
+
+
+
+    const addIphone = function (num) { 
+        let ranNum = function () {
+            let size = Math.floor(Math.random() * 100)
+
+            if (size<30) {
+               size = 30 
+            } 
+            return size
+        }
+for (let i = 0; i < num; i++) {
+    idCounter++
+    
+    let newIphone = {
+        id:  'i'+idCounter,
+        posX: Math.floor(Math.random()*80),
+        posY: Math.floor(Math.random()*80),
+        color: randomColor(),
+        size: ranNum() + 'px'
+    }
+    iphones.push(newIphone)
+    
+}
+
+     
+    }
+
+    const removeIphone = function (iphoneId) {
+        score++
+       for (let i = 0; i < iphones.length; i++) {
+           
+        if (iphones[i].id == iphoneId) {
+
+            iphones.splice(i,1)
+            
+        }
+           
+       }
+        
+    }
+   
+
+    const advanceLevel = function () {
+        score+=(timer*10)
+    
+        level++
+        if(level>10){
+            timer = level-2
+        }else{
+            timer =level
+        }
+        if (level>20) {
+            timer =level -7
+            
+        }
+        if (level>30) {
+            timer = level -12
+        }
+        if (level>40) {
+            timer = level/2
+            
+        }
+        
+        addIphone(level)
+    }
+    const decTimer = function () {
```

Comment:
sometimes naming something a short name is not always the best. Add a few more letters and make the function `const decreaseTimer` it makes your code a lot easier to read which is super important